PLAsTiCC.ipynb

```diff
@@ -306,27 +306,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import george\n",
```

Comment:
Please delete, uncomment, or explain why you commented it?

Reply:
deleted, old kernel